README.md

```diff
@@ -54,5 +54,4 @@ If you would like to contribute to this project please read the [guidelines on c
 ## Project Sponsors
 |Sponsor|Description|
 |:---:|---|
-|[![KeenIO](img/keen_logo.png)](https://keen.io/)     				      |Deliver fast, flexible analytics to your teams & customers. With Keen’s developer-friendly APIs, it’s easy to embed custom dashboards and reports in any app or website.|
-|[![TrackJS](img/trackjs_logo.png)](https://trackjs.com/)                 |Track Better Errors. Actually Fix Bugs. Know when your users run into errors and fix bugs before they report them.|
+|[![trackjs_logo](https://user-images.githubusercontent.com/10409032/52751772-a0d33a00-2fbe-11e9-922f-ab2d30e27253.png)](https://trackjs.com/)                 |Track Better Errors. Actually Fix Bugs. Know when your users run into errors and fix bugs before they report them.| 
```

Comment:
Let's fix the image URL to the original path `img/trackjs_logo.png`

Reply:
I've come across an issue where the image will appear on firefox but not on chrome when doing so but I'll change it and update the request